keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -388,6 +388,14 @@ kpxc.initLoginPopup = function() {
     const loginItems = [];
     for (let i = 0; i < kpxc.credentials.length; i++) {
         const loginItem = getLoginItem(kpxc.credentials[i], showGroupNameInAutocomplete, i);
+
+        // Ignore a duplicate entry if the password is empty, but there's already a similar entry with a password.
+        // An usual use case with TOTP in a separate database.
+        const similarEntryFound = kpxc.credentials.some(c => c.password !== '' && c.login === loginItem.login);
```

Comment:
This should be done outside of the for loop

Reply:
Should it? Then we need another loop to remove the duplicates. But sure, I can investigate if this is smarter to do before constructing the lists.

Reply:
This check doesn't depend on the current iterated item. But does it need to continuously recheck if there is a duplicate?

Reply:
Sorry ignore me, I didn't take into account the login Item value